tests/testthat/test-exercise.R

```diff
@@ -218,3 +218,27 @@ test_that("serialized exercises produce equivalent evaluate_exercise() results",
     env_vals(ex_eval_rmote$feedback$checker_args$envir_result)
   )
 })
+
+# filter_dependencies() ---------------------------------------------------
+
+test_that("filter_dependencies() excludes non-list knit_meta objects", {
+  ex <- mock_exercise(
+    user_code = deparse(quote(
+      htmltools::tagList(
+        htmltools::tags$head(htmltools::tags$style(".leaflet-container {backround:#FFF}")),
+        learnr::html_dependency_jquery()
```

Comment:
(Prolly from local copy/paste debugging)
```suggestion
        html_dependency_jquery()
```